annotate.el

```diff
@@ -173,11 +183,38 @@ major mode is a member of this list (space separated entries)."
          ,@body)
      (setf inhibit-modification-hooks t)))
 
+(defun annotate-calc-end-of-line ()
+  (save-excursion
+    (end-of-line)
+    (point)))
+
+(defun annotate-calc-beginning-of-line ()
```

Comment:
I would call this `annotate-beginning-of-line-pos`, similar to the original `beginning-of-line`, but with an added `pos` to indicate that the cursor is not moved.

Reply:
ok!